### DISCUSSION TITLE
Does `removeparam` act on URL-encoded query parameters?

### DISCUSSION THREAD

**Discussion by partingscientist** [uBlockOrigin/uBlock-issues#2683] [Q&A] [answered]
Given the following link
`https://www.sciencedirect.com/science/article/abs/pii/S0048969723030334?via%3Dihub`,

adding `||sciencedirect.com^$removeparam=via` does not remove the `via` parameter. 
Is this intended? Thank you.

**Comment by gorhill** [comment, marked as answer]
No, but for your specific case you can make it work:

    ||sciencedirect.com/*?via%3D$removeparam=/^via%3D/

**Reply by iam-py-test** [reply]
`||sciencedirect.com^$removeparam=via%3Dihub` also appears to work

**Reply by gorhill** [reply]
I am assuming the `ihub` part is subject to change.

**Reply by krystian3w** [reply]
Also possible test:

```adb
||sciencedirect.com/science/article/*?*$removeparam
```
